jmx-scraper/src/integrationTest/java/io/opentelemetry/contrib/jmxscraper/target_systems/TargetSystemIntegrationTest.java

```diff
@@ -90,32 +85,20 @@ void afterEach() {
 
   @Test
   void endToEndTest() {
+    int jmxPort = PortSelector.getAvailableRandomPort();
 
     target =
-        createTargetContainer(JMX_PORT)
-            .withLogConsumer(new Slf4jLogConsumer(logger))
+        createTargetContainer(jmxPort)
+            .withLogConsumer(new Slf4jLogConsumer(targetSystemLogger))
             .withNetwork(network)
-            .withExposedPorts(JMX_PORT)
             .withNetworkAliases(TARGET_SYSTEM_NETWORK_ALIAS);
     target.start();
 
-    String targetHost = target.getHost();
-    Integer targetPort = target.getMappedPort(JMX_PORT);
-    logger.info(
-        "Target system started, JMX port: {} mapped to {}:{}", JMX_PORT, targetHost, targetPort);
-
-    // TODO : wait for metrics to be sent and add assertions on what is being captured
-    // for now we just test that we can connect to remote JMX using our client.
-    try (JMXConnector connector = JmxConnectorBuilder.createNew(targetHost, targetPort).build()) {
-      assertThat(connector.getMBeanServerConnection()).isNotNull();
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
-
```

Comment:
[for reviewer] this part had to go anyway as it's now replaced by actual integration tests in #1485 , here it only checked the connection from the host as a proxy of "target JVM is working with JMX settings".